nucypher/crypto/constants.py

```diff
@@ -9,4 +9,4 @@
 # These lengths are specific to secp256k1
 CAPSULE_LENGTH(98)
 PUBLIC_KEY_LENGTH(33)
-PUBLIC_ADDRESS_LENGTH(42)
+PUBLIC_ADDRESS_LENGTH(20)
```

Comment:
err..? Yes ok good.

Reply:
Yeah, I'm sayin'.  I think you understand now.  :-)

Reply:
:heavy_check_mark: